restli-tools/src/main/java/com/linkedin/restli/tools/symbol/RestLiSymbolTableProvider.java

```diff
@@ -239,30 +207,6 @@ public RestLiSymbolTableProvider(Client client,
                                    long timeout,
                                    String serverNodeUri,
                                    SymbolTable responseSymbolTable)
-  {
```

Comment:
Also put this revert into PR description and CHANGELOG.

Reply:
I did?